app/views/claims/existing_session.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-panel govuk-panel--interruption">
+      <h1 class="govuk-heading-xl">
+        You have an application in progress for <%= policy_service_name(current_policy_routing_name) %>.
+      </h1>
+
+      <h2 class="govuk-heading-m">
```

Comment:
I think we just need to add govuk-!-margin-bottom-9 to force it to have a decent space between the text and the button.

Reply:
![localhost_3000_maths-and-physics_existing-session](https://user-images.githubusercontent.com/619082/72056564-c3e89580-32c4-11ea-9f99-41846219d905.png)

Duly added